tst/test_cache.py

```diff
@@ -17,5 +17,5 @@ def f():
         return pd.DataFrame({"a": [1, 2], "b": [3, 4]})
 
     for ignore_cache in [True, False]:
-        res = cache_function_dataframe(f, "f", ignore_cache=ignore_cache)
+        res = cache_function_dataframe(f, "f", cache_path="tmp_cache_dir", ignore_cache=ignore_cache)
```

Comment:
Where is `tmp_cache_dir` defined?

Reply:
I don't remember off the top of my head how to pass a tempdir as an input to a function call, feel free to send a PR if you happen to know. For now adding a TODO.